src/Sarif.Converters/Sarif.Converters.csproj

```diff
@@ -52,7 +52,7 @@
       <ItemGroup>
         <!-- We have to ship pre-patch versions of NewtonSoft for VisualStudio SDK. 
              We mitigate risk by limiting nesting depth. -->
-        <PackageReference Include="Newtonsoft.Json" Version="8.0.3" NoWarn="NU1903" />
+        <PackageReference Include="Newtonsoft.Json" Version="9.0.1" NoWarn="NU1903" />
```

Comment:
If we're taking this as part of essentially a hotfix, I'd recommend avoiding (if possible --may not be) dependency updates.  

If we do have to take this: is the comment still accurate/NoWarn still needed?  Just want to be sure.

Reply:
Yes, good catch. First, I had an unpushed release note entry on this, will resolve that. 

This upgrade to `9.0.1` is implied by the move to `net462`. The latest VS updates should allow for this (we may even be able to be more stringent). 

Push release notes update.